.rubocop.yml

```diff
@@ -215,3 +215,6 @@ Style/TrailingWhitespace:
 # Configuration parameters: EnforcedStyle, SupportedStyles.
 Style/VariableName:
   Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
Removed and fixed, https://github.com/bbaugher/confluent/pull/18/commits/2bde1faa3482fbfb3fd9c88caa1a327bad058cf0